systems/statuses.md

```diff
@@ -1,41 +1,124 @@
 # Inflicted status effects
 
+The following status effects are infliced on your character by various actions or events. These remain on the character until explicitly cleared. This will be described either in the status, or in the rule that applied the status.
+
 ## Grappled
-A grappled character may not move using the move or manouvering actions.
+
+A grappled character has been grabbed by a foe or magical effect. They cannot move without first escaping, and it is difficult to  
+
+ - A grappled character may not move using their move or manouvering actions.
+ - [Attacks](rolls.md#Attack) made against a grappled target are made with an [advantage](rolls.md#Advantage).
+
 The grappled character may use its action to attempt to escape the grapple using a strength test against the grappler.
 
-Attacks made against a grappled target are made with an [advantage](rolls.md#Advantage).
 
 ## Restrained
 Restrained is an advanced form of [grappled](#grappled). You may not be both grapped and restrained.
 
-The restrained character may not move or use any actions that require their hands, such as attacking, or casting spells with a somatic component.
-
-Attacks made against a restrained target are made with a `2x` [advantage](rolls.md#Advantage).
+ - A restrained character may not move using their move or manouvering actions.
+ - The restrained character may not use any actions that require their hands, such as [attacking](rolls.md#Attack), or casting [spells](spells.md) with a [somatic component](spells.md#Somatic).
+ - [Attacks](rolls.md#Attack) made against a restrained target are made with a `2x` [advantage](rolls.md#Advantage).
 
 The restrained character may use its action to attempt to escape being restrained using a strength test against the restrainer.
 
 ## Prone
-A prone character is lying on the ground. Their ability to move is reduced, and it is difficult to use weapons.
+A prone character is lying on the ground. Their ability to move is reduced, and it is difficult to use weapons. Your character may willingly become prone by taking half of their movement during a move action to do so.
+
+ - A prone character has its move actions modified. They may instead:

```

Comment:
Do we also want something about being prone giving disadvantage on checks/tests requiring movement?

Reply:
Thats fair - but we dont yet have rules that involve this. And may never?